sites/public/pages/listings.tsx

```diff
@@ -52,36 +33,19 @@ export default function ListingsPage(props: ListingsProps) {
       </Head>
       <MetaTags title={t("nav.siteTitle")} image={metaImage} description={metaDescription} />
       <PageHeader title={t("pageTitle.rent")} />
-      <div>
-        {openListings(props.openListings)}
-        {closedListings(props.closedListings)}
-      </div>
+      { !listingsLoading && <div>
+      <ListingsList listings={listingsData.items} />
+      <AgPagination
+        totalItems={listingsData.meta.totalItems}
+        totalPages={listingsData.meta.totalPages}
+        currentPage={currentPage}
+        itemsPerPage={itemsPerPage}
+        quantityLabel={t("applications.totalApplications")}
+        setCurrentPage={setCurrentPage}
+        setItemsPerPage={setItemsPerPage}
+      />
```

Comment:
I worry a bit about using this component that was originally set up for applications. I wonder if we will need to edit it to make it more general? Some parts (like one of the options being 1000 items per page) seem a bit more application centric.

Reply:
The component text also says "N of Total Applications"

Reply:
I'll file a ticket to make this more general. Also updated the component text to "N Total Listings"

Reply:
The component is used in the partner portal, right? So we'll just need make the text a prop?

Reply:
The text is already a prop. I think the only AI here is to change the values of `AG_PER_PAGE_OPTIONS`, which is used to display the options for # of items per page, and isn't ideal for the listings case. Because we're not using applications I think it's fine to edit this directly instead of passing it as a prop.

Reply:
Ah! sorry I missed that. Do you think we should document the places we change things so that, in the future when applications are turned back on, it's easy to make them work?

Reply:
This is on the docket to discuss in the SWE meeting today. :)